vello_hybrid/vello_api/README.md

```diff
@@ -0,0 +1,47 @@
+<div align="center">
+
+# Vello API
+
+**Public API types**
+
+[![Apache 2.0 or MIT license.](https://img.shields.io/badge/license-Apache--2.0_OR_MIT-blue.svg)](#license)
+\
+[![Linebender Zulip chat.](https://img.shields.io/badge/Linebender-%23gpu-blue?logo=Zulip)](https://xi.zulipchat.com/#narrow/stream/197075-gpu)
+[![GitHub Actions CI status.](https://img.shields.io/github/actions/workflow/status/linebender/vello/ci.yml?logo=github&label=CI)](https://github.com/linebender/vello/actions)
+
+</div>
+
+This crate defines the public API types for the Vello rendering. It provides common interfaces and data structures used across different implementations, including CPU, GPU, and hybrid rendering backends.
+
+## Features
+- Shared API types for Vello's rendering pipeline.
+- Interfaces for render contexts and rendering options.
+- Designed for compatibility across CPU and GPU implementations.
+
+## Usage
+This crate is intended to be used by other Vello components and external consumers needing a stable API.
+
+## Minimum supported Rust Version (MSRV)
+
+TODO: Fill in the MSRV when known.
```

Comment:
Is this because of the Edition 2024 question? Otherwise, I'd expect to set this to the repository MSRV by default.